src/index.html

```diff
@@ -13,6 +13,40 @@
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card card--image"
+      data-qa="card"
+    >
+      <div class="card__text">
+        <p class="card__text1">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="card__text2">Product code: 195434</p>
+        <div class="content">
```

Comment:
If "content" is a _Block_ that is child of another _Block_ called "card", the name here should also contain the parent, like so: `card__content content`